### DISCUSSION TITLE
system logs should persist reboot

### DISCUSSION THREAD

**Discussion by Ognian** [kairos-io/kairos#580] [Q&A] [answered]
I'm investigating a hardware problem (a x86 board) and found that after power down and reboot I do not have any logs from the previous boot (journalctl has as the "oldest" entry only data for the current boot)...
My first thought was that log data should be made persistent.
Or am I doing something wrong?

**Comment by mudler** [comment, marked as answer]
Good point - is this referring to systemd based system I suppose? Can you open an issue so we can track it?

**Reply by Ognian** [reply]
#597
